### DISCUSSION TITLE
Is Git server down?<title>

### DISCUSSION THREAD

**Discussion by saibalaji1997** [community/community#44485] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Stuck on "Processing your file" after uploading for a very long time.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @saibalaji1997 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
